Algorithm/Bit_manipulation/readme.md

```diff
@@ -52,3 +52,5 @@ There are different bitwise operations used in the bit manipulation. Bit operati
 
 
 ## Questions :
+
```

Comment:
Remove extra line

Reply:
@raksha009 
I have made all the necessary changes that you have told about.
Please review the changes and let me know if any further changes are required or not.
I am new to this and I am having a good time learning about these things.
Regards,
Debanjan Poddar
Fixes - #44